src/main/java/uk/nhs/tis/trainee/notifications/dto/LtftUpdateEvent.java

```diff
@@ -79,16 +80,33 @@ public record DiscussionsDto(String tpdName, String tpdEmail) {
   }
 
   /**
-   * Unpack the current status to set the state and timestamp.
+   * A DTO for state change details.
+   *
+   * @param reason  The reason for the state change.
+   * @param message A message associated with the state change.
+   */
+  @Builder
+  public record LftfStatusInfoDetailDto(String reason, String message) {
+
+  }
+
+  /**
+   * Unpack the current status to set the state, timestamp and detail.
    *
    * @param status The value of the status property.
    */
   @JsonProperty("status")
   private void unpackCurrentStatus(Map<String, Object> status) {
-    Map<String, String> current = (Map<String, String>) status.get("current");
-    state = current.get("state");
-    String timestampString = current.get("timestamp");
+    Map<String, Object> current = (Map<String, Object>) status.get("current");
+    state = (String) current.get("state");
+    String timestampString = (String) current.get("timestamp");
     timestamp = timestampString == null || timestampString.isBlank() ? null
         : Instant.parse(timestampString);
+    Map<String, String> detail = (Map<String, String>) current.get("detail");
+    stateDetail = detail == null ? null
+        : LftfStatusInfoDetailDto.builder()
+            .reason(detail.get("reason"))
```

Comment:
I am thinking for the `detail.reason`, we may need some mapping to translate the value into the human-understandable lable if the unsubmitted if done on TSS. 
Here is the mapping function of the values: https://github.com/Health-Education-England/tis-trainee-ui/blob/main/utilities/Constants.tsx#L615

Reply:
Ah, that's only on prod. On staging the abbreviation is used 🤔 . Let me look into this...

Reply:
The "Proposed start date" looks to be unsubmitted from TIS, but TSS store reason a bit differently.